framework/src/main/java/org/tron/common/overlay/server/PeerConnectionCheckService.java

```diff
@@ -41,7 +41,7 @@
 
   @PostConstruct
   public void check() {
-    logger.info("start the PeerConnectionCheckService");
+    logger.info("start the PeerConnectionCheck service");
```

Comment:
Actually there's a `PeerConnectionCheckService` class. so the original is OK.

Reply:
ok